### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gkhan205**
ReadMe updated for Code of Conduct link which was in this issue https://github.com/wlsf82/protractor-helper/issues/107.

## Pull Request Checklist

- [x] Have you followed the steps in our [Contributing](https://github.com/wlsf82/protractor-helper/blob/master/docs/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wlsf82/protractor-helper/pulls) for the same update/change?
- [x] Have you successfully ran tests with your changes locally?
- [x] Have you written new tests for your changes, if necessary?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you made improvements to the documentation, if necessary? ([docs/](https://github.com/wlsf82/protractor-helper/tree/master/docs) folder and [README.md](https://github.com/wlsf82/protractor-helper/blob/master/README.md))
- [x] Have you linked the correct [GitHub issue](https://github.com/wlsf82/protractor-helper/issues), if there is any?

> Pull requests that do not pass the coding static analysis and the automated tests on [Github Actions](https://github.com/wlsf82/protractor-helper/actions), and the code quality verification on [Better Code Hub](https://bettercodehub.com/) will not be reviewed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Many of the helper functions on this library uses `protractor.ExpectedConditions
 
 - [External resources](#external-resources)
 - [Contributing](#contributing)
+- [Code Of Conduct](#code-of-conduct)
```
Comment (wlsf82) [inline README.md:49]:
```suggestion
- [Code of conduct](#code-of-conduct)
```

File: README.md [hunk 2]
```diff
@@ -309,6 +310,10 @@ Here we list some external resources such as blog posts about the protractor-hel
 
 See [contribution guidelines](docs/CONTRIBUTING.md).
 
+## Code Of Conduct
```
Comment (wlsf82) [inline README.md:313]:
```suggestion
## Code of conduct
```

File: README.md [hunk 3]
```diff
@@ -309,6 +310,10 @@ Here we list some external resources such as blog posts about the protractor-hel
...
+
+Checkout [code of conduct](CODE_OF_CONDUCT.md).
```
Comment (wlsf82) [inline README.md:315]:
```suggestion
Checkout the [code of conduct](CODE_OF_CONDUCT.md).
```

**Comment by wlsf82** [comment]
Hi, @gkhan205, thanks for this contribution!

I've left some suggestions. Do you mind applying them, please?

Also, would you mind updating the `docs/PT-BR/README.md` file too?

In this file, the content should be:

- [Código de conduta](#código-de-conduta)

And

## Código de conduta

Veja o [código de conduta](./CODE_OF_CONDUCT.md).

**Comment by gkhan205** [comment]
> Hi, @gkhan205, thanks for this contribution!
> 
> I've left some suggestions. Do you mind applying them, please?
> 
> Also, would you mind updating the `docs/PT-BR/README.md` file too?
> 
> In this file, the content should be:
> 
> * [Código de conduta](#c%C3%B3digo-de-conduta)
> 
> And
> 
> ## Código de conduta
> Veja o [código de conduta](./CODE_OF_CONDUCT.md).

Sure will do it now. 😄

File: README.md [hunk 4]
```diff
@@ -310,9 +310,9 @@ Here we list some external resources such as blog posts about the protractor-hel
 
 See [contribution guidelines](docs/CONTRIBUTING.md).
 
-## Code Of Conduct
+## Code cf conduct
```
Comment (wlsf82) [inline README.md:313]:
```suggestion
## Code of conduct
```

Oops!

File: docs/PT-BR/README.md [hunk 5]
```diff
@@ -323,6 +324,10 @@ Listamos aqui alguns recursos externos como postagens de blog sobre o `protracto
 
 Veja as [orientações de contribuição](CONTRIBUTING.md).
 
+## Código de conduta
+
+Veja o [código de conduta](CODE_OF_CONDUCT.md)
```
Comment (wlsf82) [inline docs/PT-BR/README.md:329]:
```suggestion
Veja o [código de conduta](./CODE_OF_CONDUCT.md)
```

**Comment by wlsf82** [comment]
We're almost there, @gkhan205. Do you mind applying these last suggestions, please?

File: README.md [hunk 6]
```diff
@@ -309,6 +310,10 @@ Here we list some external resources such as blog posts about the protractor-hel
 
 See [contribution guidelines](docs/CONTRIBUTING.md).
 
+## Code of conduct
+
+Checkout the [code of conduct](CODE_OF_CONDUCT.md).
```
Comment (wlsf82) [inline README.md:315]:
```suggestion
Checkout the [code of conduct](./CODE_OF_CONDUCT.md).
```

Just one more. Sorry.

**Review by wlsf82 — APPROVED** [review]
Thanks for the contribution, @gkhan205!

Later on, I'll put you in the Credits section of the `README.md` file.
Or, if you want it, feel free to do that yourself.

You got a 🍰!

**Comment by gkhan205** [comment]
> Thanks for the contribution, @gkhan205!
> 
> Later on, I'll put you in the Credits section of the `README.md` file.
> Or, if you want it, feel free to do that yourself.
> 
> You got a 🍰!

Thank you 😄 . I will update that and create a merge request for it.

**Pull request merged**